go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/n0madic/twitter-scraper
+module github.com/dataxpe/twitter-scraper
```

Comment:
Why?

Reply:
just ignore that change. it was necessary so I could use my fork in my code.

Reply:
how can I ignore it in a pull request?